x-pack/plugins/rollup/public/crud_app/services/query_params.js

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export function extractQueryParams(queryString) {
+  if (!queryString || queryString.trim().length === 0) {
+    return {};
+  }
+
+  const extractedQueryParams = {};
+  const queryParamPairs = queryString.split('?')[1].split('&').map(paramString => paramString.split('='));
```

Comment:
Can we please use `require('query_string').parse()` instead of reimplementing query string parsing without tests?

Reply:
https://github.com/elastic/kibana/issues/24776